src/express.re

```diff
@@ -1,23 +1,27 @@
-let module Next = {
+module Next = {
   type t;
 };
 
-let module Request = {
+module Request = {
   type t;
 };
 
-let module Response = {
+module Response = {
   type t;
   external sendFile : t => string => 'a => unit = "" [@@bs.send];
```

Comment:
That's a good idea. I wanted to see if the more specific typing in this PR was the direction we wanted to go before deprecating the existing methods. I'll add a comment above that line and open a PR.